app/Grade.php

```diff
@@ -73,12 +74,12 @@ public static function getGradesForBranch($branchID)
         return $grades;
     }
 
-    private static function _gradesForBranchForSelect($branchID, $filter)
+    private static function gradesForBranchForSelect($branchID, $filter)
     {
         $grades = [];
 
-        foreach (self::_gradesForBranch($branchID, $filter) as $grade) {
-            $grades[$grade->grade] = self::mb_trim($grade->rank[$branchID]).' ('.$grade->grade.')';
+        foreach (self::gradesForBranch($branchID, $filter) as $grade) {
+            $grades[$grade->grade] = self::mbTrim($grade->rank[$branchID]).' ('.$grade->grade.')';
```

Comment:
Looking at this, I am thinking we might want to consider spaces around the concatenation operator on both sides (and have no problems with this being binary operators in general). The '.' tends to blend in too much visibly, and get lost.

Reply:
StyleCI keeps changing that back.  I may look into if I can disable that check.